Kconfig.xtos-dbg

```diff
@@ -12,3 +12,12 @@ config DEBUG_BLOCK_FREE
 	  already freed block of memory. Enabling this feature increases
 	  number of memory writes and reads, due to checks for memory patterns
 	  that may be performed on allocation and deallocation.
+
+config DEBUG_FORCE_COHERENT_BUFFER
```

Comment:
Can we have a Kconfigs here that then force all other allocation types to be uncached too i.e. from each zone

Reply:
> Can we have a Kconfigs here that then force all other allocation types to be uncached too i.e. from each zone

Do you want one item to control all zones or a set of Kconfig items with 1:1 map to each zone?